shell/browser/electron_browser_main_parts_linux.cc

```diff
@@ -115,17 +118,21 @@ std::string MaybeFixPlatformName(const std::string& ozone_platform_hint) {
 }  // namespace
 
 void ElectronBrowserMainParts::DetectOzonePlatform() {
+  auto env = base::Environment::Create();
   auto* const command_line = base::CommandLine::ForCurrentProcess();
   if (!command_line->HasSwitch(switches::kOzonePlatform)) {
-    const auto ozone_platform_hint =
+    auto ozone_platform_hint =
         command_line->GetSwitchValueASCII(switches::kOzonePlatformHint);
+    if (ozone_platform_hint.empty() &&
+        env->HasVar(kElectronOzonePlatformHint)) {
+      env->GetVar(kElectronOzonePlatformHint, &ozone_platform_hint);
+    }
```

Comment:
The `HasVar()` check isn't needed; `GetVar()` has no effect if the env var isn't set

```suggestion
    if (ozone_platform_hint.empty()) {
      env->GetVar(kElectronOzonePlatformHint, &ozone_platform_hint);
    }
```

Reply:
Good catch! I've fixed it in latest push.